_includes/glossary.html

```diff
@@ -21,7 +21,7 @@
 {%- for slug in actual -%}
   {% assign item = gloss | where: "slug", slug | first %}
   {% if item[language] %}
-  <dt id="{{item.slug}}">{{item[language].term}}</dt>
+<dt id="{{item.slug}}">{{item[language].term}} {% if language != 'en' %}<span class="in-english">(in English: {{item['en'].term}})</span>{% endif %}</dt>
```

Comment:
1. I don't know why this diff is showing up - `master` has this line and so does the `governance` branch.
2. Why are we giving special status to English?

Reply:
This line was hoovered in c9aad1005fd4af15a88a5236f221f5d6a45704f7, which is from https://github.com/carpentries/glosario/pull/56 as a potential solution to #53.